CHANGES.md

```diff
@@ -14,6 +14,7 @@
 - Parsing support has been added for unparenthesized walruses in set literals, set
   comprehensions, and indices (#2447).
 - Pin `setuptools-scm` build-time dependency version (#2457)
+- Exclude typing-extensions version 3.10.0.1 due to it being broken on 3.10 (#2460)
```

Comment:
Since these versions are nearly the same, would it make sense to specify `on CPython 3.10`?